core/src/main/java/org/elasticsearch/monitor/fs/FsService.java

```diff
@@ -33,7 +33,7 @@
  */
 public class FsService extends AbstractComponent {
 
-    private final FsProbe probe;
+    private FsProbe probe;
```

Comment:
In general I would prefer that too, but then we would face the same issue in `MonitorService`. `MonitorService` is instantiated directly in Node. I'm not sure how we should change node to allow a plugin to replace this dependency.